src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -81,4 +84,12 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  /** Returns the total number of messages for all users. */
+  public int getTotalMessageCount(){
+    Query query = new Query("Message");
```

Comment:
As I understand it, calling `Query.setKeysOnly()` on the query object will ensure that the query fetches only the keys for the entities and not the underlying data, making the query more efficient.